### ISSUE TITLE
error when try to build

### ISSUE THREAD

**Issue by thomleg51** [Atmosphere-NX/Atmosphere#139]
i download newset source but unable to compile new sartophese because i got error .
can help me please and solution without closing ?

**Comment by adamnock** [comment]
You need to stop. Your interference is not needed, wanted, or valuable in any way. Please leave and do not return

**Comment by Comput3rUs3r** [comment]
I don't think this is the place to learn to program however I think adam is being a cock. :)

**Comment by adamnock** [comment]
There’s a history with this guy making similar harassing posts and releasing ‘builds’ of the project publically

He was banned as thomleg50
And is thomhack on gbatemp, same problem

Cock level comment is warranted :)

**Comment by Comput3rUs3r** [comment]
very well then, proceed. :+1:

**Comment by simonmkwii** [comment]
You can't compile "sartophese" yet because two necessary ipc commands have not yet been implemented into libnx.

**Comment by thomleg51** [comment]
@simonmkwii how can implement ? do you make tutorial for us ?
i see on discrod @SciresM do with path " rom sf " on oddyey . how do that ?

**Comment by simonmkwii** [comment]
Are you that damn impatient?
I want romfs modding too, but you're acting as if the world depends on it.
**WAIT FOR THE OFFICIAL IMPLEMENTATION.**

P.S. I swear to god, don't you DARE @ me again thomleg51.
I will not have you clogging up my emails with your rectally dispensed "issue".
Someone close this shit.

**Comment by drofdoom** [comment]
It's not ready to compile anyways .

**Comment by foskon** [comment]
Stop being annoying and wait.
